doc/tutorials.md

```diff
@@ -96,6 +97,41 @@ lein migrate migrate
 
 migratus will use the migrations from a folder, what is configured in `config/dev/config.edn`.
 
+## Database seed with data
+
+With extending migration configuration with `seeds-dir` and `seeds-table-name` you can use
+
+```shell
+lein seed create
+lein seed migrate
+lein seed reset
+lein seed destroy
+```
+
+commands. Every defined profile can have a different seeds directory to have different dataset for different
+environments. If you're using this method to seed your data, keep your eye on the database structure is already updated
```

Comment:
"keep your eye on the database structure and make sure it is already updated when the seeding happens"